### ISSUE TITLE
Kee Firefox extension no longer connecting, settings claims "unable to manage" connections

### ISSUE THREAD

**Issue by Terrance** [kee-org/keepassrpc#135]
I'm using KeePass and Kee for Firefox, and both were working fine yesterday, but today (with no system changes that I'm aware of) Kee is reporting KeePass is not open.  After some poking around KeePassRPC I found this warning in the _Authorised clients_ tab:

> Sorry, you're unable to manage the connections you have authorised. You can probably fix the problem by (re-)installing the latest version of the Microsoft .NET framework.

I've installed via Chocolately (through the `keepass` and `keepass-rpc` packages), which do not appear to pull in any .NET Framework dependencies (or any dependencies for that matter) -- the only related program I see installed is the .NET Core runtime, which I've reinstalled but has made no difference.  I've not been prompted for any Windows updates since this was all working, so I'm not sure what else would have changed here.

**Comment by gazd25** [comment]
I'm also seeing this exact same issue on two of my devices, no changes to the KeePass or Kee plugin.

I noticed it by accident because my timeout expired on a previous authorisation, so wanted to check all of them and was greeted with the same message as Terrance on the KeePassRPC plugin Authorised Clients menu.

I then went and checked and it appears that the same has also occurred on my other windows computer.

.Net is all up to date on both so unsure where the issue lies.
